sinopia_client/docs/LDPApi.md

```diff
@@ -74,6 +75,7 @@ Name | Type | Description  | Notes
 ------------- | ------------- | ------------- | -------------
  **slug** | **String**| The suggested URI path for the group. | 
  **group** | [**LDPContainer**](LDPContainer.md)| Group metadata to insert into base container and describe the group. | 
+ **link** | **String**| specifies container type.  you probably shouldn&#39;t override this parameter for this operation. | [optional] [default to &lt;http://www.w3.org/ns/ldp#BasicContainer&gt;; rel&#x3D;&quot;type&quot;]
```

Comment:
good catch!  hadn't even thought about that being unnecessary encoding.

so all the documentation (other than comments in the swagger.yml and the spec implementations), including this markdown, is totally auto-generated by swagger-codegen.  it'll get re-generated as the spec changes and we update the SDK accordingly.

i don't personally find these docs all that useful, so i think i'd prefer to just ignore them, or remove them and document usage as needed in the wiki or a readme or something.  my least favorite option is editing them, since i suspect there won't be much payoff (because i sorta doubt people will reference them much).

this documentation is moderately useful in that it gives example calls, and it requires no extra work at all (which maintaining a wiki would, even if minimal).  so it has that going for it as an argument to not just purge it.  but i wouldn't feel bad about killing it and leaving it out.